src/status_im/ui/screens/communities/community.cljs

```diff
@@ -196,7 +196,7 @@
       [blank-page (i18n/label :t/welcome-community-blank-message)]
       [list/flat-list
        {:key-fn                       :chat-id
-        :content-container-style      {:padding-bottom 8}
+        :content-container-style      {:padding-bottom 8 :margin-top -8}
```

Comment:
could you elaborate pls?

Reply:
Hi @flexsurfer,
`community-chat-list` had a top margin of 8 by default(PR Summary Screenshot). It seemed it was being added by flat list. So instead of changing the component, I just overrode the container style.

Reply:
The issue was with the header. It had a bottom padding. And I thought flat-list has a default margin 😂😅.
Removed 🤘